diff --git a/node_modules/prettier-plugin-jsdoc/lib/parser.js b/node_modules/prettier-plugin-jsdoc/lib/parser.js
index 85b9e5d..3b98730 100644
--- a/node_modules/prettier-plugin-jsdoc/lib/parser.js
+++ b/node_modules/prettier-plugin-jsdoc/lib/parser.js
@@ -12,7 +12,7 @@ const stringify_1 = require("./stringify");
 /** @link https://prettier.io/docs/en/api.html#custom-parser-api} */
 exports.getParser = (parser) => function jsdocParser(text, parsers, options) {
     const ast = parser(text, parsers, options);
-    if (!options.jsdocParser) {
+    if (!options || !options.jsdocParser) {
         return ast;
     }
     /**
